### ISSUE TITLE
Add properties for modulation type and subchannels to BroadcastFrequencySpecification

### ISSUE THREAD

**Issue by vholland** [schemaorg/schemaorg#2111]
In some cases, it is not enough to provide a broadcast frequency to describe a radio station. One must also know the sub channel. I propose two new properties:

- ```broadcastSignalModulation``` to specify things like AM vs FM
- ```broadcastSubChannel``` to specify the subchannel

An example would be:
```
{
  "@context": "http://schema.org/",
  "@type": "BroadcastService",
  "broadcastFrequency": {
    "@type": "BroadcastFrequencySpecification",
    "broadcastFrequencyValue": "89.3",
    "broadcastSignalModulation": "FM",
    "broadcastSubChannel": "HD1"
  }
}

```

**Comment by thadguidry** [comment]
+1 is SubChannel a fairly global term or standard here ? or is there a better international term ?

**Comment by vholland** [comment]
I am not a domain expert, but is the term I found in reading up on the subject

**Comment by danbri** [comment]
+1 sounds plausible to me

ping @njh @rvguha w.r.t. naming choices

**Comment by vholland** [comment]
Implemented in PR #2144

**Comment by vholland** [comment]
Included in schema.org 3.5.
